README.md

```diff
@@ -1,92 +1,99 @@
-# [Lodex](http://lodex.inist.fr) &middot; [![Build Status](https://travis-ci.org/Inist-CNRS/lodex.svg?branch=master)](https://travis-ci.org/Inist-CNRS/lodex) [![bitHound Overall Score](https://cdn.rawgit.com/aleen42/badges/master/src/gitbook_1.svg)](https://lodex.gitbooks.io/lodex-user-documentation) [![Licence](https://img.shields.io/badge/licence-CeCILL%202.1-yellow.svg)](http://www.cecill.info)
+[![Lodex CI](https://github.com/Inist-CNRS/lodex/actions/workflows/lodex-ci.yml/badge.svg)](https://github.com/Inist-CNRS/lodex/actions/workflows/lodex-ci.yml)
+[![Licence](https://img.shields.io/badge/licence-CeCILL%202.1-yellow.svg)](http://www.cecill.info)
+[![Documentation](https://img.shields.io/badge/Documentation-f48022)](https://lodex.inist.fr/docs/)
+![Docker Image Version (latest semver)](https://img.shields.io/docker/v/inistcnrs/lodex)
+
+# [Lodex](http://lodex.inist.fr)
 
 <img src="https://user-images.githubusercontent.com/7420853/30152932-1794db3c-93b5-11e7-98ab-a7f28d0061cb.png" width=150 align=right>
 
-Lodex is a tool to enable publishing a set of data `csv`, `tsv`, `xml`, `json`, ... in semantic web formats `JSON-LD`, `N-Quads`, ... and propose to manipulate them in a backoffice.
+Lodex is a tool facilitating the publication of a dataset in various formats, including 'csv', 'tsv', 'xml', 'json', among others.
+The platform also offers features to manipulate the data in a back-office environment.
 
-To see what Lodex can do, check out <https://data.istex.fr/> or the user documentation at <https://lodex.gitbook.io/lodex-user-documentation/>
+To see what Lodex can do, please refer to the website at https://data.istex.fr/ or 
```

Comment:
```suggestion
To see what Lodex can do, please refer to the website at <https://data.istex.fr/> or 
```

Please use [markdownlint](https://github.com/DavidAnson/markdownlint#overview), to ensure Markdown will work on most Markdown engine (I know it works as is in GitHub, but you never know...).